chicago/permit_cleaning.py

```diff
@@ -301,28 +367,63 @@ def save_xlsx_files(df, max_rows, file_base_name):
     df_review_empty_invalid.to_excel(file_name_review_empty_invalid, index=False, engine="xlsxwriter")
 
 
+if __name__ == "__main__":
```

Comment:
Most of the diff that follows represents indentation changes due to the addition of the `if __name__ == "__main__"` conditional block here. I'll call out semantic changes via comment.